src/mcts/node.cc

```diff
@@ -160,9 +162,59 @@ float Edge::GetP() const {
   return ret;
 }
 
+void Edge::MakeTerminal(GameResult result) {
+  certainty_state_ |= kTerminalMask | kCertainMask | kUpperBound | kLowerBound;
+  certainty_state_ &= kGameResultClear;
+  if (result == GameResult::WHITE_WON) {
+    certainty_state_ |= kGameResultWin;
+  } else if (result == GameResult::BLACK_WON) {
+    certainty_state_ |= kGameResultLoss;
+  }
+}
+
```

Comment:
Run clang-format, it will fix long lines and double empty lines between functions.